private-dns-config.yml

```
@@ -10,7 +10,7 @@ subscriptions:
     resourceGroup: core-infra-intsvc-rg
 
   - name: "DTS-CFTPTL-INTSVC"
-    zoneTemplate: '${["plum","probate","sscs","idam","ccd","ts","hmc","aac","rpo","camunda","dg","dm","dtsse","em","et","sptribs","nfdiv","pcq","lau","wa","finrem","jps","rd","xui", "reform-scan","bulk-scan","rpe","fact","rpts","am","ia","bar","fees","fees-register","ccpay","payment","fis","div"].contains(product) || ["draft-store-service","div-cos","civil-general-applications", "civil-service", "civil-ccd","civil-camunda","civil-citizen-ui","civil-ga-ccd","rpe-service-auth-provider", "rpe-pdf-service", "cmc-citizen-frontend", "nfdiv-case-api", "nfdiv-frontend","prl-ccd-definitions","prl-cos","prl-dgs","prl-citizen-frontend","prl-ccd-case-migration","adoption-cos-api","adoption-web","ccd-case-disposer","fpl-case-service","bar-web"].contains(appFullName) ? environment + ".platform.hmcts.net" : "service.core-compute-" + environment + ".internal"}'
+    zoneTemplate: '${["plum","probate","sscs","idam","ccd","ts","hmc","aac","rpo","camunda","dg","dm","dtsse","em","et","sptribs","nfdiv","pcq","lau","wa","finrem","jps","rd","xui", "reform-scan","bulk-scan","rpe","fact","rpts","am","ia","bar","fees","fees-register","ccpay","payment","fis","div"].contains(product) || ["draft-store-service","div-cos","civil-general-applications", "civil-service", "civil-ccd","civil-camunda","civil-task-configuration","civil-citizen-ui","civil-ga-ccd","rpe-service-auth-provider", "rpe-pdf-service", "cmc-citizen-frontend", "nfdiv-case-api", "nfdiv-frontend","prl-ccd-definitions","prl-cos","prl-dgs","prl-citizen-frontend","prl-ccd-case-migration","adoption-cos-api","adoption-web","ccd-case-disposer","fpl-case-service","bar-web"].contains(appFullName) ? environment + ".platform.hmcts.net" : "service.core-compute-" + environment + ".internal"}'
```

can it just be all of civil ?
```suggestion
    zoneTemplate: '${["plum","probate","sscs","idam","ccd","ts","hmc","aac","rpo","camunda","dg","dm","dtsse","em","et","sptribs","nfdiv","pcq","lau","wa","finrem","jps","rd","xui", "reform-scan","bulk-scan","rpe","fact","rpts","am","ia","bar","fees","fees-register","ccpay","payment","fis","div","civil"].contains(product) || ["draft-store-service","div-cos","rpe-service-auth-provider", "rpe-pdf-service", "cmc-citizen-frontend", "nfdiv-case-api", "nfdiv-frontend","prl-ccd-definitions","prl-cos","prl-dgs","prl-citizen-frontend","prl-ccd-case-migration","adoption-cos-api","adoption-web","ccd-case-disposer","fpl-case-service","bar-web"].contains(appFullName) ? environment + ".platform.hmcts.net" : "service.core-compute-" + environment + ".internal"}'
```

---

I believe it can be as previously we only had few of the services, but now we are adding all. I have commited your suggestion. Thanks